modules/restheart/scripts/src/db.converters.ts

```diff
@@ -1,28 +1,20 @@
-/**
- * Created by nitzan on 02/01/2017.
- */
-
-/// <reference path="../../../../scripts/bin/app/application.d.ts" />
-/// <reference path="../../../../scripts/bin/components/components.d.ts" />
-/// <reference path="../../../../scripts/bin/components/converters.d.ts" />
-
-(function(): void {
-	fugazi.components.modules.descriptor.loaded({
+(function (): void {
+	fugazi.loaded({
 		name: "restheart.databases",
 		converters: {
 			RawDatabases2Databases: {
 				title: "converts RawDatabases to Databases",
 				input: "RawDatabases",
 				output: "Databases",
-				converter: function(response: { _embedded: { _id: string }[] }): string[] {
+				converter: function (response: { _embedded: { _id: string }[] }): string[] {
```

Comment:
(that specific file was removed as it was in `modules/restheart`, I'm addressing the general issue)
regarding additional whitespaces, I hear you. agreed, it adds noise that reduces the ability to review.
I've used the IDE's auto-format. tracking down these spaces by hand can be a lot of manual work. if there's a de-facto style rule we should keep, can we express it in a machine-friendly format so that the computer will take care of it?